src/chainstate/burn/db/sortdb.rs

```diff
@@ -1748,37 +1748,37 @@ impl<'a> SortitionHandleConn<'a> {
         &self,
         prepare_end_bhh: &BurnchainHeaderHash,
         pox_consts: &PoxConstants,
+        first_block_height: u64,
```

Comment:
Isn't this in `self.context` already?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2233/commits/fd507c59944e9f7d373b364b2beceea6cf00315a